sonar-plugin-api/src/main/java/org/sonar/api/ce/measure/MeasureComputer.java

```diff
@@ -142,6 +144,11 @@
        */
       void addMeasure(String metric, String value);
 
+      /**
+       * Return list of issues of current component.
+       */
+      List<? extends Issue> getIssues();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove usage of generic wildcard type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1452)